app/controllers/api/v3/sync_controller.rb

```diff
@@ -69,15 +69,12 @@ def params_with_errors(params, errors)
   def capture_errors(params, errors)
     return unless errors.present?
 
-    Raven.capture_message(
-      "Validation Error",
-      logger: "logger",
```

Comment:
This logger attributes go away - we weren't really using this in with the old sentry anyways.